### PR TITLE
Move some more traits to Mods.Common

### PR DESCRIPTION
**Pull request by reaperrr**
Moves the following to Common:
- Production and related files
- the rest of OrderGenerators
- the rest of non-RA Render traits

### CODE REVIEW

File: OpenRA.Mods.RA/OpenRA.Mods.RA.csproj [hunk 1]
```diff
@@ -132,22 +131,14 @@
...
     <Compile Include="Scripting\Properties\GuardProperties.cs" />
     <Compile Include="Widgets\Logic\TabCompletionLogic.cs" />
-    <Compile Include="Production.cs" />
-    <Compile Include="ProductionBar.cs" />
     <Compile Include="Traits\Render\RenderJammerCircle.cs" />
-    <Compile Include="Render\RenderBuildingWarFactory.cs" />
-    <Compile Include="Render\RenderHarvester.cs" />
-    <Compile Include="Render\RenderDisguise.cs" />
-    <Compile Include="Render\RenderLandingCraft.cs" />
+    <Compile Include="Traits\Render\RenderHarvester.cs" />
```
Comment (penev92) [inline OpenRA.Mods.RA/OpenRA.Mods.RA.csproj]: Both TD and RA use this. You know what to do ;)

File: OpenRA.Mods.Common/OpenRA.Mods.Common.csproj [hunk 2]
```diff
@@ -260,6 +267,9 @@
     <Compile Include="Traits\Power\Power.cs" />
     <Compile Include="Traits\Power\RequiresPower.cs" />
     <Compile Include="Traits\Power\ScalePowerWithHealth.cs" />
+    <Compile Include="Traits\Production.cs" />
+    <Compile Include="Traits\ProductionBar.cs" />
```
Comment (penev92) [inline OpenRA.Mods.Common/OpenRA.Mods.Common.csproj]: This should not be here. I was going to suggest Render or Graphics, (was leaning toward the latter), but I see there is `TimedUpgradeBar` in `Traits\Render`, which is actually a perfect example ;)

File: OpenRA.Mods.RA/OpenRA.Mods.RA.csproj (hunk 1 above)
Comment (reaperrr) [inline OpenRA.Mods.RA/OpenRA.Mods.RA.csproj]: Yes, but this requires the ```Harvester``` trait, and you said I shouldn't touch that stuff yet, so I just moved it inside Mods.RA to get rid of the old ```Render``` folder ;)

Comment (penev92) [inline OpenRA.Mods.RA/OpenRA.Mods.RA.csproj]: Good point.

**Comment by penev92** [comment]
:+1:

**Comment by Mailaender** [comment]
Needs a rebase.

**Comment by reaperrr** [comment]
Rebased.

**Pull request merged**